source/Jobbr.Storage.MsSql.Tests/Jobbr.Storage.MsSql.Tests.csproj

```diff
@@ -23,31 +23,25 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.4.0" />
     <PackageReference Include="MSTest.TestAdapter" Version="2.2.10" />
     <PackageReference Include="MSTest.TestFramework" Version="2.2.10" />
-  </ItemGroup>
-  <ItemGroup>
-    <Reference Include="Microsoft.CSharp" />
+    <PackageReference Include="Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers" Version="0.4.355802">
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>

```

Comment:
Oh, missed that before. Should also be removed

Reply:
I actually looked for it an just didn't see it.